backend/preprocess.py

```diff
@@ -39,6 +40,23 @@ def second_vote(mandates: List[Mandate]) -> List[Mandate]:
     return sort.second_vote(filtered_mandates)
 
 
+def second_vote_results(constituency: str, party: str) -> int:
+    if (second_vote_results := SECOND_VOTE_RESULTS.get(constituency)) != None:
+        return second_vote_results[party]
+    else:
+        return ""
+
+
+def constituency(constituency: str) -> str:
+    if "(" in constituency:
+        index = constituency.find("(")
```

Comment:
These two lines can be combined. `.find` return `-1` (or `0`, please check) if it can't find the pattern.

Reply:
if constituency.find("(") != -1:
    index = constituency.find("(")

Is there another solution to make it a one liner? I can't come up with a solution to make it a one liner

Reply:
There is one, but let's merge with the current one. I can show you how to do it later.